components/02-elements/action/Action.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div class="action">
+    <div
+      class="action__handler"
+    >
+      <alpaca-button
+        v-if="button"
+        :class="[customClass, 'action__button']"
+        @click="click"
```

Comment:
Not directly related to this component, but general rule

It's better to emit this event outside component, than passing the function inside (I know it's an React thing to do it this way)

So it should look like this

```vue
@click="onClick"
...
methods: {
  onClick(event) {
    this.$emit('click', event)
  }
}
```
and then in the parent component

```vue
<child @click="someEventHandler" />
```